joplin/base/models/topic_page.py

```diff
@@ -63,6 +63,3 @@ class TopicPageTopicCollection(ClusterableModel):
     panels = [
         PageChooserPanel('topiccollection'),
     ]
-
-    def __str__(self):
-        return self.topiccollection.text
```

Comment:
this field dosen't exist and makes checking relationships in the shell impossible

Reply:
Is this the only thing you changed? I can't tell with the linting. Can you turn off linting on your PRs until we get whole app on the same page?